### ISSUE TITLE
Materialized views are not include in database info

### ISSUE THREAD

**Issue by egor-bayandin** [sasha-alias/sqltabs#33]
Hi!
I found issue, that materialized view is not showing in database info page.
Can you fix it pls.

I think it's because that information_scheme [doesn't contain information about it](http://www.postgresql.org/message-id/CAH7T-ao6ece1mgCsCvsE04W59ZZJP9gGXVK97wkUzRV5gsDqQQ@mail.gmail.com), but it's possible to get this list using following query:
```sql
SELECT oid::regclass::text
FROM   pg_class
WHERE  relkind = 'm';
```

Example what i saw now:
![2015-11-10 17-01-51 sql tabs](https://cloud.githubusercontent.com/assets/10337429/11065656/a8c6498a-87d5-11e5-8873-d9951a40da37.png)
and what i got using upper select:
![2015-11-10 18-06-34 sql tabs](https://cloud.githubusercontent.com/assets/10337429/11065691/da446aa0-87d5-11e5-968f-2fd55d41ff94.png)

**Comment by sasha-alias** [comment]
Yes, MVs are not supported yet, they are in the awaiting list of features.

**Comment by egor-bayandin** [comment]
ok, thank you!
